src/theme/utils.ts

```diff
@@ -77,11 +77,25 @@ export const getColorErrors = [
 ];
 
 /**
- *
  * @param figmaTokensObject The parsed objects from Figma Tokens in the src/theme/constants/ dir
- * @returns the value of the figma token item converted into rem
+ * @param type 'pixels' refers to all the string values that need to be converted to rem
+ *             'string' refers to all the string values that need no conversion (e.g. opacity, font-family etc)
+ *             'number' refers to all the string values that need to be converted to numbers (e.g. font-weight)
  */
 export const getFigmaTokensValue =
```

Comment:
this is a great addition, i will update the type so we can avoid all the code to double define what is returned in every use